codegen/core/src/main/scala/com/lightbend/akkasls/codegen/Packaging.scala

```diff
@@ -58,9 +58,15 @@ object FullyQualifiedName {
     FullyQualifiedName(
       descriptor.getName,
       serviceType match {
+        // View and Actions services clashes with akka-grpc service generation
+        // therefore we need to append [View,Action] to it (or Impl).
+        // FIXME: should we find a better solution for that?
         case ServiceType.SERVICE_TYPE_VIEW =>
           if (descriptor.getName.endsWith("View")) descriptor.getName + "Impl"
           else descriptor.getName + "View"
+        case ServiceType.SERVICE_TYPE_ACTION =>
+          if (descriptor.getName.endsWith("Action")) descriptor.getName + "Impl"
+          else descriptor.getName + "Action"
```

Comment:
This is kind of weird and can be surprising for users. I will create a ticket for this. We may want to revisit it and improve later.

Reply:
it was added in https://github.com/lightbend/akkaserverless-java-sdk/pull/208 (but only for views)